features/src/rapids-build-utils/opt/rapids-build-utils/bin/update-build-dir-links.sh

```diff
@@ -46,6 +46,7 @@ update_build_dir_links() {
             for ((j=0; j < ${!py_length:-0}; j+=1)); do
                 local py_sub_dir="${repo}_python_${j}_sub_dir";
                 local py_path=~/"${!repo_path:-}${!py_sub_dir:+/${!py_sub_dir}}";
+                # TODO: Should this also preserve the _skbuild directory for scikit-build projects?
```

Comment:
We don't need to do anything here, because we can't use a custom build dir unless the project uses `scikit-build-core`. If no custom build dir, no need to set up symlinks.